### DISCUSSION TITLE
Display book's title instead of chapter title

### DISCUSSION THREAD

**Discussion by ezekiel747** [PaulWoitaschek/Voice#1900] [Ideas]
**Is your feature request related to a problem? Please describe.**
in the books list i'm seeing the first chapter title instead of the actual book title (i've imported the list of books with the option for audiobooks folder)
Screenshot below.

**Describe the solution you'd like**
see the book title instead (taken from the actual folder name containing that specific book)

**Describe alternatives you've considered**
-

**Additional context**
-
![Screenshot_20211028_081753](https://user-images.githubusercontent.com/38600480/139192122-ec8aa5eb-7a53-4560-9fbf-46be1d024829.jpg)

**Comment by Christedge** [comment]
As a heavy user of getAbstract, I want the title always being available in all views and the search, so as to save a lot of time when searching for a particular audio book.

Acceptance criteria

- The list view shows the title of an .mp3 (e.g. instead of the artist or album) consistently to the drop down control.
- The details view shows the title of an .mp3 (e.g. instead of the artist or album) consistently to the drop down control.
- The title tag is included in the search.

Below you can see how getAbstract tags its .mp3. In the list and detail view, it is barely impossible to recognize which file is which book.

![AudiobookPlayerDropdown1](https://user-images.githubusercontent.com/33388286/226414226-1d90a4fe-6c5e-49f4-a33c-1db1207f664c.jpg)
![AudiobookPlayerDropdown2](https://user-images.githubusercontent.com/33388286/226414234-1ed4a649-4f19-4b10-9cb5-9c72f8d79bd3.jpg)
![AudiobookPlayerViewDetail](https://user-images.githubusercontent.com/33388286/226414254-06ca53e4-a404-47a2-ad1d-81e5fea7b078.jpg)
![AudiobookPlayerViewList](https://user-images.githubusercontent.com/33388286/226414260-c59c9b20-5ed9-4e82-9a97-d3ad1fef999d.jpg)
![qoobar](https://user-images.githubusercontent.com/33388286/226414270-8cb84a4e-c949-46f5-ae3e-0d800f7b4f14.png)
